dir.targets

```diff
@@ -17,7 +17,30 @@
 
             var tempFile = Path.Combine(directory, Path.GetRandomFileName());
             var client = new System.Net.WebClient();
-            client.DownloadFile(Address, tempFile);
+            var tryCount = 1;
+            var maxTries = 3;
+
+            while (tryCount <= maxTries)
+            {
+                try
+                {
+                    Log.LogMessage("Attempting to download {0}...", Address);
+                    client.DownloadFile(Address, tempFile);
+                    break;
+                }
+                catch (System.Net.WebException e)
```

Comment:
I didn't figure it was worth distinguishing as the other sorts of errors shouldn't come up often and will get logged anyway. Leaning towards simpler given that it is an inline task.